apps/microtvm/reference-vm/README.md

```diff
@@ -72,20 +72,20 @@ For example:
 $ ./base-box-tool.py --provider virtualbox build zephyr
 ```
 
-2. **Run** release tests for each platform:
+2. **Run** release tests for each microTVM device:
```

Comment:
I think that 'platform' here actually refers to PLATFORM (like `zephyr` or `arduino`), so not related to MICROTVM_PLATFORM. Hence that's an occurrence of 'platform' that is correct and should not be replace in the context of the proposed change.

Reply:
as this is in context of a base box release test, i think this should use the same language as we use in tests/micro/zephyr/test_zephyr.py

Reply:
revert to platform.

Reply:
LGTM!